conf/uppmax.config

```diff
@@ -77,6 +91,15 @@ if (hostname.startsWith("i")) {
   params.config_profile_description = 'UPPMAX (Irma) cluster profile provided by nf-core/configs.'
```

Comment:
Should we also remove the Irma config?

Reply:
that would make sense but perhaps leave it in at this moment since the cluster is still available for a week or so 😄

Reply:
ok!